db/migrate/20240311053739_rename_stripe_transaction_to_stripe_record.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class RenameStripeTransactionToStripeRecord < ActiveRecord::Migration[7.1]
+  def change
+    rename_table :stripe_transactions, :stripe_records
+    rename_column :stripe_webhook_events, :stripe_transaction_id, :stripe_record_id
+    rename_column :stripe_payment_intents, :stripe_transaction_id, :stripe_record_id
+
+    reversible do |direction|
+      direction.up do
+        execute <<-SQL
+          UPDATE registration_payments
+          SET receipt_type = 'StripeRecord'
+          WHERE receipt_type = 'StripeTransaction'
+        SQL
```

Comment:
You can do this without manually writing SQL strings by executing `RegistrationPayment.update_all(receipt_type: 'StripeRecord')`

Reply:
Ah of course! We don't need to have any `WHERE` logic because we only have `StripeTransaction`s at present

Reply:
Actually, you can refine this approach: `RegistrationPayment.where(receipt_type: 'StripeTransaction').update_all(receipt_type: 'StripeRecord')`